HashPlayer.java

```diff
@@ -1,23 +1,58 @@
-//
 import java.util.*;
-public class Player {
+
+public class HashPlayer implements PlayerInterface {
 	// Valid is Static so All Instances: Player 1 and Player 2 Share the Variable

```

Comment:
Nit - all instances of HashPlayer, not any Player, share instances
Also not good practice to comment on how static works in a codebase